products/template/build_scripts/transform.py

```diff
@@ -21,12 +46,21 @@ def intermediate():
     _run_step("historic_landmarks")
 
 
-def outputs():
+def product():
     _run_step("templatedb")
-    _run_step("aggregation/boroughs")
+    _run_step("aggregation/templatedb_boroughs")
+
+
+def test():
+    _run_dbt_step(["deps"])
+    _run_dbt_step(["debug"])
+    _run_dbt_step(["test", "--select", "source:*"])
+    _run_dbt_step(["test", "--select", "templatedb"])
+    _run_dbt_step(["test", "--select", "aggregation"])
 
 
 if __name__ == "__main__":
     staging()
```

Comment:
Not sure if we need `staging()` if all it does is creates copies of source tables. Thoughts?

Reply:
true it doesn't do much, but it does a couple of valuable things: selects the subset of columns that are actually useful and renames some to be either more readable or more standardized

see [docs](https://docs.getdbt.com/best-practices/how-we-structure/2-staging#staging-models) on ideas behind the value of using staging models